src/fiatconnect/ReviewScreen.tsx

```diff
@@ -44,11 +45,32 @@ export default function FiatConnectReviewScreen({ route, navigation }: Props) {
   const { flow, normalizedQuote, fiatAccount } = route.params
   const providerId = normalizedQuote.getProviderId()
 
+  const fiatConnectQuotesError = useSelector(fiatConnectQuotesErrorSelector)
   const fiatConnectQuotesLoading = useSelector(fiatConnectQuotesLoadingSelector)
   const fiatConnectQuotes = useSelector(fiatConnectQuotesSelector)
   const [showingExpiredQuoteDialog, setShowingExpiredQuoteDialog] = useState(false)
   const [usingUpdatedQuote, setUsingUpdatedQuote] = useState(false)
 
+  useEffect(() => {
+    if (!fiatConnectQuotesLoading && usingUpdatedQuote && fiatConnectQuotesError) {
```

Comment:
What if there is not an error fetching fiatconnect quotes but theres no `updatedQuoteData` because quote.ok is false?

Reply:
Handled that now